logging/config/cluster-logging-external.adoc

```diff
@@ -1,25 +1,30 @@
 :context: cluster-logging-external
 [id="cluster-logging-external"]
-= Sending logs to external devices using Fluentd Forward plug-ins
+= Sending logs to external locations
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} cluster logging allows you to configure the Fluentd *out_forward* plug-in to send logs to external devices. 
+{product-title} cluster logging allows you to send logs to destinations outside of your {product-title} cluster.
+
+* *Sending logs using Fluentd*. You can use the Fluentd *out_forward* plug-in to securely send logs to an external destination that understands the Fluent forward protocol. 
+
+* *Sending logs using syslog*. The Fluentd `syslog` plug-in sends logs to another logging collector using the syslog protocol (RFC 3164). Many logging collectors such as Fluentd, Rsyslog, and others support this protocol.
 
 You can use the xref:../../logging/config/cluster-logging-log-forwarding.adoc#cluster-logging-log-forwarding[log forwarding feature], which can be easier to configure than the plugins. Note that the log forwarding feature is currently in Technology Preview.
 
 [IMPORTANT]
 ====
-Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap to configure *out_forward*, as described below, instead of editing the `secure-forward.conf` section in the `fluentd` ConfigMap. You can add any certificates required by your external devices to a secret, called `secure-forward`, which is mounted to the Fluentd Pods.
+Changes introduced by the new log forwarding feature modified the support for Fluentd *syslog* plug-in starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure the Fluentd *syslog* plug-in. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed. Before upgrading cluster logging, you can copy your current `secure-forward.conf` section and use the copied data when you create the `secure-forward` ConfigMap. 
```

Comment:
A little mess, let me instruduce the history.
To enable rsyslog 
     From 3.x to 4.2,  Add Environment variables to fluentd pods
       (For example: oc set env ds/fluentds USE_REMOTE_SYSLOG=true REMOTE_SYSLOG_HOST=syslog.example.com REMOTE_SYSLOG_TYPE=syslog)
         From 4.3.0 -> 4.3.5,  as a workaround you can set syslog configuration in secure-forward configmap.
     From 4.3.7,  you can set syslog configuration in configmap syslog.
To enable fluentd forward
    From 3.x -> 4.2.  you need to update the fluentd configmap directly.  the data file secure-forward.conf (http://pastebin.test.redhat.com/846797).  
    From 4.3.  set secure-forward configuration in configmap secure-forward.